airflow/cli/commands/scheduler_command.py

```diff
@@ -38,11 +38,24 @@ def _create_scheduler_job(args):
     return job
 
 
+def _run_scheduler_job(args, is_daemon):
+    skip_serve_logs = args.skip_serve_logs
+    job = _create_scheduler_job(args)
+    if not is_daemon:
+        signal.signal(signal.SIGINT, sigint_handler)
```

Comment:
NIT: can we move the signal handling out to the "else"  in "scheduler()" ? seems that the handlers can be set before entering `_run_scheduler_job` and this way we will be able to remove `is_daemon` param,

Reply:
ok

Reply:
Done.